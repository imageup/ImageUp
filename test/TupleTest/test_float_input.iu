function main() -> int
{
    tuple : x = (1.0, 2.0);
    printf(x[0]);
    printf(x[1]);
    prints("bituple correct output: 1, 2");

    tuple : x = (4.0, 5.0, 6.0);
    printf(x[0]);
    printf(x[1]);
    printf(x[2]);
    prints("trituple correct output: 4, 5, 6");
    return 0;
}
