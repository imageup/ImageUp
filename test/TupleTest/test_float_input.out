llc -relocation-model=pic test.ll > test.s ;gcc -o test.exe test.s lib.c `pkg-config --cflags --libs opencv`;./test.exe
1
2
bituple correct output: 1, 2
4
5
6
trituple correct output: 4, 5, 6
